src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -66,6 +72,32 @@ public GridStorageCache(final IGrid g) {
     public void onUpdateTick() {
         this.itemMonitor.onTick();
         this.fluidMonitor.onTick();
+
+        this.itemBytesTotal = 0;
+        this.itemBytesUsed = 0;
+        try {
+            for (ICellProvider icp : this.activeCellProviders) {
+                if (icp.getClass() == Class.forName("appeng.tile.storage.TileDrive")) {
```

Comment:
I'm not sure that calculating used bytes every tick is a good idea
At least move Class.forName out of the cycle

Reply:
(Looks like my comment was a bit late)

Reply:
Inside mod it can be simple replaced with `TileDrive.class`